File: src/test/java/com/lucasia/ginquiry/controller/AllControllersSmokeTest.java
package com.lucasia.ginquiry.controller;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
public class AllControllersSmokeTest {

    @Autowired
    private BrandCrudController brandController;

    @Autowired
    private GinCrudController ginController;

    @Autowired
    private UserCrudController userController;

    @Test
    public void contextLoads() throws Exception {

        Assertions.assertNotNull(brandController);

        Assertions.assertNotNull(ginController);

        Assertions.assertNotNull(userController);
    }

}

File: src/test/java/com/lucasia/ginquiry/controller/AllControllersSmokeTest.kt
package com.lucasia.ginquiry.controller;

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
class AllControllersSmokeTest {

    @Autowired
    private val brandController: BrandCrudController? = null

    @Autowired
    private val ginController : GinCrudController? = null

    @Autowired
    private val userController : UserCrudController? = null

    @Test
    fun contextLoads() {
        Assertions.assertNotNull(brandController)
        Assertions.assertNotNull(ginController)
        Assertions.assertNotNull(userController)
    }
    
}
